src/styles/card.scss

```diff
@@ -0,0 +1,128 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: (1, 200px);
```

Comment:
The value for 'grid-template-columns' seems to be incorrect. It should be a valid CSS value defining the size of the columns. For example, 'grid-template-columns: repeat(1, 200px);' if you want to define a single column layout. Make sure to follow the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).